Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -660,3 +639,35 @@ internal extension CFURLSessionEasyCode {
         throw self
     }
 }
+
+#if os(Android)
+extension URLSession {
+    fileprivate static var _sslCertificateAuthorityFilePathPtr: UnsafeMutablePointer<Int8>?
+    /// You can bypass SSL peer verification altogether (not advised)
+    public static var verifyPeerSSLCertificate = true
+    /// See https://curl.haxx.se/docs/sslcerts.html
+    /// For SSL to work you need a "cacert.pem" to be accessible at the path
+    /// provided by setting URLSession.sslCertificateAuthorityFile to a URL.
+    /// Downloadable here: https://curl.haxx.se/docs/caextract.html
+    public static var sslCertificateAuthorityFile: URL {
+        set(value) {
+            verifyPeerSSLCertificate = true
```

Comment:
Yes, I recall. I'm not arguing for a more complicated API. What I'm saying is, since you've chosen the path of presenting two properties, they should be independent of each other. This is consistent with user expectation both in Foundation and in cURL. Precisely _because_ this is a niche API that will not be amply documented, it's important that these expectations are met. It is very surprising that...

```swift
NSSession.verifyPeerSSLCertificate = false
NSSession.sslCertificateAuthorityFile = url
```

...gives a different result than...

```swift
NSSession.sslCertificateAuthorityFile = url
NSSession.verifyPeerSSLCertificate = false
```

...keeping in mind that the default for `verifyPeerSSLCertificate` is `true`, so if a user wants that setting to be `false`, he or she will necessarily have to set that value somewhere explicitly.